docs/guided-tour.md

```diff
@@ -75,16 +83,17 @@ note[MetaSections]: workflow `main` is missing both `meta` and `parameter_meta`
    = fix: add both the `meta` and `parameter_meta` sections
 
 warning[UnusedInput]: unused input `color`
-   ┌─ example.wdl:35:16
+   ┌─ example.wdl:30:16
    │
-35 │         String color = "green"
+30 │         String color = "green"
    │                ^^^^^
 ```
 
 Specific lint rules can be ignored with multiple invocations of the `-e` flag.
+Rule matching is case insensitive from command line arguments.
 
 ```shell
-sprocket lint example.wdl -e ContainerUri -e MetaSections
```

Comment:
This doesn't seem like an improvement to me. Though the CLI can accept case insensitive rules, the rules themselves are still capitalized in the output messages. So I feel it makes sense to keep them as they appear in the messages.